trackintel/geogr/distances.py

```diff
@@ -212,21 +212,22 @@ def check_gdf_crs(gdf, transform=False):
     >>> check_gdf_crs(triplegs, transform=False)
     """
     if_planer = False
-    if gdf.crs is None:
-        # projection is not defined
+    if gdf.crs is None:  # projection is not defined
         if_planer = False
```

Comment:
this seems unnecessary as it was just initialized as False

Reply:
I would keep this and other `is_planar = False` statements for readability. 

Another option is to delete the initialization (Line 214), as each branch defines is_planar, but this also hurts a bit the readability

Reply:
👍